yt/loaders.py

```diff
@@ -1650,7 +1651,9 @@ def load_sample(
     - Corresponding sample data live at https://yt-project.org/data
 
     """
-    loadable_path = _get_sample_data(fn, progressbar=progressbar, timeout=timeout)
+    loadable_path, kwargs = _get_sample_data(
```

Comment:
I would slightly prefer we use a different variable name for the output `kwarg` here (to avoid confusion when debugging)

Reply:
sure! how about `load_kwargs` ?

Reply:
works for me